### ISSUE TITLE
Bnorm layer at inference time

### ISSUE THREAD

**Issue by AruniRC** [vlfeat/matconvnet#192]
At inference time, do we remove the Batch normalization layers from the network like for dropout?

 As given in the function documentation, `vl_nnbnorm() ` calculates the batch mean and variance over "the batch packed in the 4D tensor X" of its inputs. 

However, at _inference time_, we would require the variance and mean to be calculated from the population and remain fixed instead of recomputing it every time (if my understanding of the Batch Normalization paper by Ioffe et al. is correct [Sec 3.1]). This would mean adding another layer  that applies the linear transformation of batch-normalization using supplied values of population mean and variance. 

I did not see any code for doing this in the release of 1.0-beta12. Shouldn't this be part of the example code for using a bnorm trained network? Please let me know if I have missed something.

thanks,
Aruni

**Comment by AruniRC** [comment]
Saw the code in `vl_simplenn()` which needs to be supplied the population mean and covariance to do this. Solved!
